grafana/provisioning/dashboards/chain.json

```diff
@@ -395,7 +395,78 @@
   "style": "dark",
   "tags": ["ligithning-network"],
   "templating": {
-    "list": []
+    "list": [
+      {
+        "current": {
+          "tags": [],
+          "text": "Prometheus",
+          "value": "Prometheus"
+        },
+        "hide": 0,
+        "includeAll": false,
+        "label": null,
+        "multi": false,
+        "name": "datasource",
+        "options": [],
+        "query": "prometheus",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "type": "datasource"
+      },
+      {
+        "allValue": null,
+        "current": {
+          "tags": [],
+          "text": "",
+          "value": ""
+        },
+        "datasource": "$datasource",
```

Comment:
Don't we need to set this `$datasource` value for every existing panel we have? Rn it's hardset to "Prometheus". 

Tried this out by find-and-replacing
`"datasource": "Prometheus" -> "datasource": "$datasource"` ?

Reply:
yes, you're right. Have updated.